docs/translations-needed.md

```diff
@@ -252,16 +252,8 @@ If you add or update a translation run `grunt check-translations` to update the
 |                         | , opens subtitles settings dialog                                                   |
 |                         | , opens descriptions settings dialog                                                |
 |                         | , selected                                                                          |
-| ru.json (missing 14)    | Close Modal Dialog                                                                  |
-|                         | Descriptions                                                                        |
-|                         | descriptions off                                                                    |
-|                         | Audio Track                                                                         |
+| ru.json (missing 6 )    | Close Modal Dialog                                                                  |
 |                         | The media is encrypted and we do not have the keys to decrypt it.                   |
-|                         | Play Video                                                                          |
-|                         | Close                                                                               |
-|                         | Modal Window                                                                        |
-|                         | This is a modal window                                                              |
-|                         | This modal can be closed by pressing the Escape key or activating the close button. |
 |                         | , opens captions settings dialog                                                    |
 |                         | , opens subtitles settings dialog                                                   |
 |                         | , opens descriptions settings dialog                                                |
```

Comment:
Same as the `caption settings dialog`, though, I'm not sure we're currently showing it in any case right now but it would be equivalent to the captions setting dialog when we would do it.